git-remote-hg

```diff
@@ -265,7 +265,7 @@ class Parser:
         return (user, int(date), -tz)
 
 def fix_file_path(path):
-    path = os.path.normpath(path)
+    path.replace('//', '/')
```

Comment:
You have to set `path = path.replace('//', '/')`. Otherwise the replaced string isn't saved.